samples/msal-node-samples/client-credentials-distributed-cache/src/ProvisionHandler.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { createServer, IncomingMessage, Server, ServerResponse } from "http";
+import { CryptoProvider } from "@azure/msal-node";
+import open from "open";
+
+class ProvisionHandler {
+    private server: Server;
+    private cryptoProvider: CryptoProvider;
+    private state: string;
+
+    constructor() {
+        this.cryptoProvider = new CryptoProvider();
+    }
+
+    /**
+     * Opens a browser window to grant admin consent to the application. This is required
+     * to provision the application into another tenant. For more information, visit:
+     * https://learn.microsoft.com/azure/active-directory/develop/single-and-multi-tenant-apps
+     *
+     * @param instance
+     * @param tenantId
+     * @param clientId
+     * @param scope
+     */
+    async grantAdminConsent(instance: string, tenantId: string, clientId: string, scope: string): Promise<boolean> {
```

Comment:
Hmm, our other samples do not showcase this. Do we really expect app developers to do this?

Reply:
I wanted an easy way to provision this to different tenants when testing, to properly illustrate the cache partitioning with respect to tenant Ids.I have no strong opinions about this though -happy to remove it if you think it's a concern (the [dotnet multi-tenant sample](https://github.com/Azure-Samples/ms-identity-aspnet-daemon-webapp) uses a similar approach -a web app to interact with the admin user)